Cargo.toml

```diff
@@ -1,5 +1,4 @@
 [patch.crates-io]
-h2 = { git = "https://github.com/gila/h2", branch = "v0.2.7"}
```

Comment:
Does it mean that the problem with authority header has been fixed? Or perhaps rust h2 library tolerates invalid value now (as the patch did that)?

Reply:
Hmm did we have a reproduction test I could check with? I didn't see a failure.

Reply:
Ah -- Maybe! I see some failures!